packages/kit/src/runtime/app/forms.js

```diff
@@ -161,14 +165,28 @@ export function enhance(form_element, submit = () => {}) {
 		let result;
 
 		try {
+			const is_valid_enctype = enctype?.match(
+				/application\/x-www-form-urlencoded|multipart\/form-data|text\/plain/
+			);
+
+			const headers = new Headers({
+				accept: 'application/json',
+				'Content-Type': is_valid_enctype ? enctype : 'application/x-www-form-urlencoded',
+				'x-sveltekit-action': 'true'
+			});
+
+			// @ts-expect-error
+			const body = enctype === 'multipart/form-data' ? form_data : new URLSearchParams(form_data);
```

Comment:
```suggestion
			const body = enctype === 'multipart/form-data' ? form_data : new URLSearchParams(form_data);
```
Do we need the `@ts-expect-error` here? It seems that URLSearchParams is a valid body type https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#body